initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh

```diff
@@ -185,7 +185,10 @@ setup_mounts() {
   mkdir -p $REQKICK_DIR
   mkdir -p $BUILD_DIR
 
-  REQPROC_MOUNTS="-v $BASE_DIR:$BASE_DIR"
+  REQPROC_MOUNTS="-v $BASE_DIR:$BASE_DIR \
```

Comment:
should be envs on top of this file. individual functions can append to the same variable if required

Reply:
REQPROC_MOUNTS is defined as `export REQPROC_MOUNTS=""` at the top already. Are you saying this should be:
REQPROC_MOUNTS="$REQPROC_MOUNTS -v ..."?

Reply:
yes

Reply:
Updated this and a few other places.